README.md

```diff
@@ -1,13 +1,13 @@
 # prebuild-install [![Build Status](https://travis-ci.org/mafintosh/prebuild-install.svg?branch=master)](https://travis-ci.org/mafintosh/prebuild-install)
 
-A command line tool for easily install prebuilds for multiple version of node/iojs on a specific platform.
+Command line tool to easily install prebuilds for multiple version of node/iojs
 
-`prebuild-install` supports installing prebuilt binaries from GitHub by default.
+`prebuild-install` supports installing prebuilt binaries from GitHub by default
```

Comment:
the full stop is correct here as this is a full sentence and not part of an enumeration